simpleclient_dropwizard/src/main/java/io/prometheus/client/dropwizard/samplebuilder/SampleBuilder.java

```diff
@@ -0,0 +1,23 @@
+package io.prometheus.client.dropwizard.samplebuilder;
```

Comment:
I don't see a need to split this into subdirs

Reply:
Removed impl subdir